gh-badges/lib/make-badge.js

```diff
@@ -21,58 +21,56 @@ templateFiles.forEach(async filename => {
   const extension = path.extname(filename).slice(1)
   const style = filename.slice(0, -`-template.${extension}`.length)
   // Compile the template. Necessary to always have a working template.
-  templates[`${style}-${extension}`] = dot.template(templateData)
-  if (extension === 'svg') {
```

Comment:
I changed the key in the array from ``${style}-${extension}`` to `style` since the extensions are now all svg. I also removed the if statement.